packages/e2e/.env.example

```
@@ -11,40 +11,38 @@ KEY_MANAGEMENT_PARAMS='{"bip32Ed25519": "CML", "accountIndex": 0, "chainId":{"ne
 
 # Providers setup - required by getWallet
 ASSET_PROVIDER=http
-ASSET_PROVIDER_PARAMS='{"baseUrl":"http://localhost:4000/asset"}'
+ASSET_PROVIDER_PARAMS='{"baseUrl":"http://localhost:4000/v1.0.0/asset"}'
```

Shouldn't this match what we have in [GH workflow definition](https://github.com/input-output-hk/cardano-js-sdk/pull/872/files#diff-33e5806dbe086222ebce154c323030b5a434ee370b7de1e0bf1c3d04584f28f7R10)?
```suggestion
ASSET_PROVIDER_PARAMS='{"baseUrl":"http://localhost:4000/"}'
```

---

Yes, it should. Nice catch. Let me check better this.

---

I confirm I changed only my `.env` file and I forgot to update `.env.example` accordingly.

083994d31 should solve this.

---

Looks good, thanks. Maybe we should create a follow-up task to do sth like
```
PROVIDER=http
PROVIDER_PARAMS={...}
```
to further simplify e2e .env. WDYT?

---

Created LW-8181